### PR TITLE
[WIP] demo about versioning config

### PR DESCRIPTION
**Pull request by superyyrrzz**
Related: https://github.com/OfficeDev/office-js-docs-reference/pull/285

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e6d75aa](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/e6d75aa7d27e14c9492be4e52a4c609e11f0e326)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e6d75aa](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/e6d75aa7d27e14c9492be4e52a4c609e11f0e326)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[e6d75aa](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/e6d75aa7d27e14c9492be4e52a4c609e11f0e326)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.gitignore](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/.gitignore) | :white_check_mark:Succeeded | |
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/outlook_1_4/demo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_4/demo.md) | :white_check_mark:Succeeded | |
[docs/outlook_1_4/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_4/toc.yml) | :white_check_mark:Succeeded | |
[docs/outlook_1_5/demo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_5/demo.md) | :white_check_mark:Succeeded | |
[docs/outlook_1_5/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_5/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C10%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901100408049065-675%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3BQa7Mb67S55ZDdggdqZUr8eYmCTUk4BAwE5OX6TEzI%3D&st=2019-01-10T04%3A04%3A52Z&se=2019-02-10T04%3A09%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docfx.json [hunk 1]
```diff
@@ -30,6 +44,16 @@
         ]
       }
     ],
+    "groups": {
```
Comment (superyyrrzz) [inline docs/docfx.json]: `groups` names should match `moniker_ranges` in `.openpublishing.publish.config.json`

**Comment by o365devx** [comment]
OPS Build status updates of commit _[7b24ead](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/7b24ead2b502a50c8c255c9dea1451cb2bef9720)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[7b24ead](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/7b24ead2b502a50c8c255c9dea1451cb2bef9720)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[7b24ead](https://github.com/OfficeDev/office-js-docs-pr/pull/675/commits/7b24ead2b502a50c8c255c9dea1451cb2bef9720)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[.gitignore](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/.gitignore) | :white_check_mark:Succeeded | |
[.openpublishing.publish.config.json](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[docs/docfx.json](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/docfx.json) | :white_check_mark:Succeeded | |
[docs/outlook_1_4/demo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_4/demo.md) | :white_check_mark:Succeeded | |
[docs/outlook_1_4/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_4/toc.yml) | :white_check_mark:Succeeded | |
[docs/outlook_1_5/demo.md](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_5/demo.md) | :white_check_mark:Succeeded | |
[docs/outlook_1_5/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/renzeyu/docs/outlook_1_5/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201901180646036389-675%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=s4QYHdQq7vWE9cqZH0vIp2B%2FWwGp3Ph4Ui512TInTOM%3D&st=2019-01-18T06%3A42%3A50Z&se=2019-02-18T06%3A47%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request closed without merging**